src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div
+      data-qa="card"
+      class="product-card"
+    >
+
+      <div class="product-item product-item--position">
+
+        <img
+          src="./images/imac.jpeg"
+          alt="Apple Imac"
+          class="product-item__photo"
+        >
+
+        <p class="product-item__name product-item__name--position">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+
+        <p class="product-item__code product-item__code--position">Код товара: 195434</p>
+
+      </div>
+
+      <div class="product-rating">
+
+        <div class="product-rating__stars product-rating__stars--selected">
```

Comment:
```suggestion
        <div class="product-rating__stars product-rating__stars--4">
```
Тут должно быть число, определяющее количество активных звезд в блоке